src/Logary.Tests/Registry.fs

```diff
@@ -78,12 +79,39 @@ let tests = [
     do! logm.shutdown ()
   }
 
+  testCase "log with scope" <| fun _ ->
+    let dataSlot = DataSlot.useDefault ()
+    let (logm, out, _)  = Utils.buildLogManagerWith (fun r -> r |> Config.middleware (Middleware.fillWithContextSlot dataSlot) ) |> run
+
+    for _ in 1..10 do
+
+      let loggername = PointName.parse "logger.test"
+      let lg1 = logm.getLogger loggername
+
+      let s1 = dataSlot.push (fun _ -> "scope-1")
+      lg1.infoWithBP (eventX "scope1") |> run
```

Comment:
I'm afraid we can't use 'run' in the unit tests or they might deadlock. Could you make `testCase` a `testCaseJob`?

Reply:
done.